dockers/docker-ptf/Dockerfile.j2

```diff
@@ -139,6 +139,17 @@ RUN ln -s /usr/bin/tcpdump /usr/sbin/tcpdump
 
 RUN mkdir -p /var/log/supervisor
 
+# add dependencies to build gnmi_get
+RUN wget -O go.tgz https://dl.google.com/go/go1.12.linux-amd64.tar.gz \
+    && tar -C /usr/local -xzf go.tgz \
+    && export PATH=$PATH:/usr/local/go/bin \
+    && go version \
+    && mkdir $HOME/go \
+    && export GOPATH=$HOME/go \
+    && go env \
+    && go get github.com/jipanyang/gnxi/gnmi_get \
```

Comment:
why from jipan?

Reply:
gnmi_get \
  -xpath_target CONFIG_DB
  -xpath "/system/openflow/agent/config/datapath-id" \
  -xpath "/system/openflow/controllers/controller[name=main]/connections/connection[aux-id=0]/state/address" \
  -target_addr localhost:10161 \
  -target_name hostname.com \
  -key client.key \
  -cert client.crt \
  -ca ca.crt \
  -username foo \
  -password bar \
  -alsologtostderr


Sonic gnmi has special parameter "xpath_target" which is not available in official tool. SO the community has been using the private version so far. We can propose to back port to master, which will take time.  I think before that, we can use this private branch instead of forking the source code to sonic telemetry repro.